### ISSUE TITLE
Re-number episodes

### ISSUE THREAD

**Issue by ChristinaLK** [carpentries-incubator/hpc-intro#9]
I'd be a fan of re-numbering by 5s so that it's easier for future users of this material to slot in their own pages.

**Comment by psteinb** [comment]
can you elaborate on this in light of the repo split into hpc-shell and hpc-intro?

**Comment by tkphd** [comment]
Doing so would cause `make lesson-check` to consistently throw errors, [*e.g.*](https://travis-ci.com/tkphd/hpc-intro)
```
Missing or non-consecutive episode numbers [0, 11, 12, 13, 14, 15, 16]
```
IMO, firming up the lesson plan and re-numbering consecutively from one, with placeholders for content under development, would be more helpful.

**Comment by psteinb** [comment]
but in our case, that is quite needed. We want the HPC carpentry lessons be ready to mix and match. In other words, the episodes here would go with numbers in `[10,20]`. the lessons in [hpc-shell](https://github.com/hpc-carpentry/hpc-shell/tree/gh-pages/_episodes) go with numbers in `[0,10)`. We should play with `lesson-check` to make it pass and yet keep the freedom that people can easily copy over the episodes from one repo to another. Any suggestions anyone?

**Comment by tkphd** [comment]
It seems to me that "mix & match" is handled in the other Carpentries curricula by having coupled sessions, rather than shuffled ones: we start with UNIX Shell in the morning, then cover Version Control in the afternoon, for example, rather than injecting Bash lessons into the Git material. This helps to focus attention on just one complex concept.

Dividing the workshop into 1) a subset of hpc-shell followed by 2) a subset of hpc-intro makes sense, but as delineated mini-workshops rather than an integrated jumble.

Be that as it may, from the `lesson_check.py` perspective, renumbering from [01,07] to [10,16] ought to work.

**Comment by ChristinaLK** [comment]
I guess I'm okay with letting that error happen and/or turning that check off, but the way things are now, I'd also be okay if the lessons start from 01 and are consistently numbered.

**Comment by psteinb** [comment]
I am reopening this issue to hear what other's think: @jstaf @ocaisa @aturner-epcc et al ?

**Comment by aturner-epcc** [comment]
I think the error is the inflexibility of 'make check' here rather than the numbering scheme we have chosen.

@tkphd It is not a case of losing focus on a topic; there may be extra information that needs to be inserted for a particular run of the lesson (specific to the system or the community) and this gives the flexibility to re-use the material with your own custom bits added if you want. I think we need this flexibility here.

**Comment by reid-a** [comment]
Closed, superseded by #230.
